src/registrar/models/user.py

```diff
@@ -406,3 +411,10 @@ def on_each_login(self):
     def is_org_user(self, request):
         has_organization_feature_flag = flag_is_active(request, "organization_feature")
         return has_organization_feature_flag and self.has_base_portfolio_permission()
+
+    def get_user_domain_ids(self, request):
+        """Returns either the domains ids associated with this user on UserDomainRole or Portfolio"""
```

Comment:
This may not reflect what would actually happen in production, but what would the expected outcome be if a user was a portfolio admin and also was domain manager of other domains outside of that portfolio? Or would this not be a likely case since each user can only be assigned one portfolio for now anyway?

Reply:
Makes sense and thank you for linking the slack thread! Agree that handling a user with domains in and outside a portfolio would make more sense being in another PR's scope